app/controllers/api/v1/batch_assignment_overrides_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::BatchAssignmentOverridesController < SharedSecretAuthenticatedApiController
+  include CorsSupport
+
+  def create
+    BatchArbitraryAssignmentCreation.create! create_params.merge(force: true)
+    head :no_content
+  end
+
+  private
+
+  def create_params
+    params.permit(:visitor_id, assignments: [:split_name, :variant, :mixpanel_result, :context])
```

Comment:
i left the `context` at the per-assignment-override level in the event that someone wants to provide that value per split.

if you think it should be top-level like the `visitor_id` i'm fine with that too.

also, i noticed that the original assignment endpoint has the `visitor_id` in the POST body and not in the url, so i kept that the same here. `POST /v1/batch_assignment_override`.

i'm open to naming suggestions. we already have a concept of a `BulkAssignmentCreation` which is focused on assigning a bunch of visitors to the same variant of a single split. whereas this code is assigning a single visitor a specific variant (per split) for a bunch of splits.

Reply:
deprecate the old endpoint and call this `V2:AssignmentOverridesController`? I don't see a need for the old one since this one can easily be swapped in.

i don't recall why we put visitor_id in the body. i do prefer the other way though, so if you take the suggestion of moving to v2, then i'd advocate for making that change too.